sedas_pyapi/sedas_api.py

```diff
@@ -43,7 +43,8 @@ class SeDASAPI:
     def __init__(self, username: str, password: str) -> None:
         self._username = username
         self.__password = password
-        self.login()
+        
```

Comment:
Can you tidy these extra blank lines up please. They make no difference to the code but cause linters to get cranky.